core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -46,16 +52,53 @@ oppia.factory('ThreadDataService', [
 
     var _fetchThreads = function(successCallback) {
       var fPromise = $http.get(_THREAD_LIST_HANDLER_URL);
-      var sPromise = $http.get(_SUGGESTION_LIST_HANDLER_URL, {
-        params: {
-          list_type: 'all',
-          has_suggestion: true
-        }
-      });
+      var sPromise;
+      if (constants.USE_NEW_SUGGESTION_FRAMEWORK) {
```

Comment:
Recommend you use the same pattern you use later on in the code:

Set `params` differently in the if statement, then pass `params` into `$http.get` outside the if statement.

Reply:
Done thanks!